trakt/users.py

```diff
@@ -132,27 +132,28 @@ def get_items(self):
             item_data = item.pop(item_type)
             extract_ids(item_data)
             if item_type == 'movie':
-                self._items.append(Movie(item_data['title'], item_data['year'],
-                                         item_data['slug']))
+                extract_ids(item_data)
+                movie = Movie(**item_data)
+                self._items.append(movie)
             elif item_type == 'show':
-                self._items.append(TVShow(item_data['title'],
-                                          item_data['slug']))
+                extract_ids(item_data)
+                show = TVShow(**item_data)
+                self._items.append(show)
             elif item_type == 'season':
                 show_data = item.pop('show')
                 extract_ids(show_data)
-                season = TVSeason(show_data['title'], item_data['number'],
-                                  show_data['slug'])
+                title = show_data.pop('title')
+                season = TVSeason(title, item_data['number'], **show_data)
                 self._items.append(season)
             elif item_type == 'episode':
                 show_data = item.pop('show')
                 extract_ids(show_data)
-                episode = TVEpisode(show_data['title'], item_data['season'],
-                                    item_data['number'],
-                                    show_id=show_data['trakt'])
+                extract_ids(item_data)
+                episode = TVEpisode(show_data['title'], **item_data, show_id=show_data['trakt'])
                 self._items.append(episode)
             elif item_type == 'person':
-                self._items.append(Person(item_data['name'],
-                                          item_data['slug']))
+                extract_ids(item_data)
+                self._items.append(Person(**item_data))
 
         yield self._items
```

Comment:
I noticed that `UserList.get_items()` has a bug, it always appends items to `self._items`. IMHO it should reset `self._items = []` on each call to `get_items()`.